### PR TITLE
Add variable Value at Risk threshold mu and tests

### LINKED ISSUE #1525: [CVaR]: Add VaR threshold variable
### Description

- Create the variable VaR threshold variable ($\mu$) at confidence level $\alpha$. Use a descriptive name `value_at_risk_threshold_mu`
  - Add a new file for this variable here https://github.com/TulipaEnergy/TulipaEnergyModel.jl/tree/main/src/variables
  - Double-check if we actually need to add the corresponding SQL query: https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/src/sql/create-variables.sql
  - Only create the new variable if the value of the $\lambda$ parameter (see #1524) is greater than zero
- The variable is continuous and positive (double check for upper bound)
- It doesn't depend on any index, and the units are the currency of the objective function
- Add a test that checks that the variable is created when the $\lambda$ parameter is greater than zero

### PR DESCRIPTION
**Pull request by florinelaseur**
Add variable Value at Risk threshold mu and tests

Closes #1525 

## Checklist

- [ ] I am following the [contributing guidelines](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/docs/src/90-contributing/90-contributing.md)
- [ ] Tests are passing
- [ ] Lint workflow is passing
- [ ] Docs were updated and workflow is passing

### CODE REVIEW

File: src/sql/create-variables.sql [hunk 1]
```diff
@@ -499,3 +499,18 @@ where
...
+;
+
+create table var_value_at_risk_threshold_mu as
+select
+    nextval('id') as id,
+    cast(null as float8) as solution,
+;
+
+drop sequence id
+;
```
Comment (datejada) [inline src/sql/create-variables.sql]:
```suggestion
```
